llvm-external-projects/iree-dialects/include/iree-dialects/Dialect/LinalgExt/IR/LinalgExtBase.td

```diff
@@ -99,4 +99,24 @@ def EncodingAttr :
   let genVerifyDecl = 0;
 }
 
+def RequiresSyncAttr : IREELinalgExt_Attr<"RequiresSync"> {
```

Comment:
Have you considered to add the attribute to https://github.com/openxla/iree/tree/main/compiler/src/iree/compiler/Codegen/Dialect/Codegen/IR ?

We are planing to collapse LinalgExt into IREE main tree. It will be put to https://github.com/openxla/iree/tree/main/compiler/src/iree/compiler/Dialect

This seems not a good fit for the attribute because this is codegen specific.

https://groups.google.com/g/iree-discuss/c/E2EpE6nFPHw/m/reTX0I-5AwAJ